README.md

```diff
@@ -1,24 +1,263 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* ...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 EL Dorado Backend <a name="about-project"></a>
+![eldorado-logo](https://user-images.githubusercontent.com/84629565/211169801-59cc4f73-65db-4784-81db-c591b74b87ce.png)
+
+**El Dorado Backend** is a luxary car rental application where the user can register new accounts, login and reserve a set of different cars. It is built and connected by using two different repos, including Back-end(Rails) and Front-end(React/Redux).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Luxary Cars List endpoint]**
+- **[Devise authentication to access Reservations]**
+- **[JWT Authorization upon login and register]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- API Docs -->
+
+## <img src="https://emojipedia-us.s3.amazonaws.com/source/microsoft-teams/337/spiral-notepad_1f5d2-fe0f.png" width="23" height="20"/> API Documentation<a name="api-docs"></a>
+
+- Here is the API documentation of the project [El Dorado API-Docs](https://eldorado.onrender.com/api-docs/)
+![api-docs](https://user-images.githubusercontent.com/84629565/212746269-ce103339-03ab-4ef6-aa5e-fea72dd7c378.png)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ER DIAGRAM-->
+## :card_index: Entity Relationship Diagram <a name="er-diagram"></a>
+![er-diagram](https://user-images.githubusercontent.com/84629565/212745874-93da1ede-2e1e-49d8-9593-e5475d56e5b6.png)
+
+<!-- LIVE DEMO -->
+## <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="23" height="20"/> React Frontend <a name="react-frontend"></a>
+- Here is the Frontend part of the project [El Dorado Frontend](https://github.com/michael-duke/ElDorado-Frontend)
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] :smiley: not yet available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+In order to run this project you need:
+`Git` and `Ruby`
+```
+ gem install rails
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+
+Create Database (Mandatory)
+
+```
+rails db:create
+```
+
+If you have made a migration then run this command
+
+```
+rails db:migrate
+```
+```
+rails db:seed
```

Comment:
- [ ] These are mandatory steps to run your app, which are included in your PR description (I appreciate it):

![image](https://user-images.githubusercontent.com/57421823/212815808-e65f85f7-0ac5-47a1-b75b-02e28df43dec.png)

Since these steps are mandatory, kindly add them to your readme, so everybody (including potential employers) can setup and run your app successfully. 👍

Reply:
We will add this change, thank you for pointing it out @Meltrust

Reply:
## 👍 
<img width="312" alt="image" src="https://user-images.githubusercontent.com/84629565/212835129-81e3f201-c18e-4527-9e12-16a9116270b3.png">